sgr_project/StopDistanceData sddata
---
float32 stopdistance